samples/cpp/create_mask.cpp

```diff
@@ -123,14 +123,16 @@ void mouseHandler(int event, int x, int y, int, void*)
 
 int main(int argc, char **argv)
 {
-
-    if(argc != 2)
+    cv::CommandLineParser parser(argc, argv, "{@input||}");
+    if (!parser.has("@input"))
     {
-        cout << "usage: " << argv[0] << " <input_image>" << endl;
-        exit(1);
+        cout << "usage: " << parser.getPathToApplication() << " <input_image>" << endl;
```

Comment:
`getPathToApplication` returns path to folder where application is located, not the program name.